src/spells.cpp

```diff
@@ -911,7 +911,7 @@ bool InstantSpell::playerCastInstant(Player* player, std::string& param)
 		}
 	} else if (hasParam) {
 		if (getHasPlayerNameParam()) {
-			Player* playerTarget = nullptr;
+			std::shared_ptr<Player> playerTarget = nullptr;
```

Comment:
Could this be `const auto`
```suggestion
			auto playerTarget = std::shared_ptr<Player>(nullptr);
```